geomesa-accumulo/geomesa-accumulo-spark-runtime/pom.xml

```diff
@@ -81,9 +81,7 @@
         <dependency>
             <groupId>org.locationtech.geomesa</groupId>
             <artifactId>geomesa-accumulo-datastore_${scala.binary.version}</artifactId>
-            <version>${project.version}</version>
             <classifier>tests</classifier>
-            <scope>test</scope>
```

Comment:
Did this dependency move to compile?

Reply:
no, we were just declaring it in a weird way - i standardized it into the main pom.xml

Reply:
Ah! ok!